### PR TITLE
Pipeline, Sergey Mozgovoy - 21FPL2

### PR DESCRIPTION
**Pull request by Sergo-M**

### CODE REVIEW

**Review by dmitry-uraev — COMMENTED** [review]
Done with score 6

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -44,37 +86,61 @@ def __init__(self, lemma: str = "", pos: str = "", tags: str = ""):
...
+        pos = self._morphological_parameters.pos
+        xpos = '_'
+        if include_morphological_tags and self._morphological_parameters.tags:
+            feats = self._morphological_parameters.tags
+        else:
+            feats = '_'
+        head = '0'
+        deprel = 'root'
+        deps = '_'
+        misc = '_'
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pipeline.py]: i would set these in return statement directly

File: lab_6_pipeline/pipeline.py [hunk 2]
```diff
@@ -144,16 +254,72 @@ def __init__(self, corpus_manager: CorpusManager):
...
+                                                         tags=self._converter.convert_morphological_tags(tags=token['analysis'][0]['gr']))
+                    conllu_token.set_morphological_parameters(morph_params)
+                    conllu_tokens.append(conllu_token)
+                elif 'analysis' in token:
+                    token_position += 1
+                    conllu_token = ConlluToken(text=token['text'], position=token_position)
+                    morph_params = MorphologicalTokenDTO(lemma=token['text'],
+                                                         pos='X')
+                    conllu_token.set_morphological_parameters(morph_params)
+                    conllu_tokens.append(conllu_token)
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pipeline.py]: all elifs can be simplified - look at similar lines

**Review by dmitry-uraev — COMMENTED** [review]
Good job!

File: lab_6_pipeline/pipeline.py (hunk 1 above)
Comment (Sergo-M) [reply lab_6_pipeline/pipeline.py]: Done

File: lab_6_pipeline/pipeline.py (hunk 2 above)
Comment (Sergo-M) [reply lab_6_pipeline/pipeline.py]: Done

**Comment by dmitry-uraev** [comment]
Good job!

**Pull request closed without merging**